README.md

```diff
@@ -179,6 +179,15 @@ Branch returns explicit parameters every time. Here is a list, and a description
 | +clicked_branch_link | Denotes whether or not the user clicked a Branch link that triggered this session
 | +click_timestamp | Epoch timestamp of when the click occurred
 
+##### Initialization tips and tricks
+
+Very important thing is to call Branch.InitSession(...) at the start of your app else Branch has not time to registry callback and you will receive nothing.
```

Comment:
Certain place is first scene method Start() in any enabled script.
We don't need to update InitSession location in our testbed, because we have one scene, we have enabled BranchDemo.cs that calls Start() at the start of the app.

Our partner called InitSession after 3 seconds and the following occurs:
- application:didFinishLaunchingWithOptions
- application:openURL:sourceApplication:annotation
- InitSession(callback)

in our testbed the following occurs:
- InitSession(callback)
- application:didFinishLaunchingWithOptions
- application:openURL:sourceApplication:annotation